cvxpy/atoms/integrate/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Copyright, the CVXPY Ashok Viswanathan
```

Comment:
```suggestion
Copyright, the CVXPY Project
```

Please sign the CLA.